test/fixtures/injector.js

```diff
@@ -20,9 +19,5 @@ module.exports = function () {
     'runtime/'
   ]);
 
-  ioc.registerDependencies({
-    colors: colors
-  });
```

Comment:
Is this the only place it was used?

Reply:
If so we will need to publish this as a breaking change. Since the expectation of this injectable/published dependency will go away.